### ISSUE TITLE
Secure supertenant and tie up other security considerations

### ISSUE THREAD

**Issue by steveellis** [culibraries/folio#153]
Think we'll want to do this before we start putting data in.  There is a secure-supertenant.py file floating around out there but I can't put my hands on it.

I'd also like to figure out the following:
* How should we handle module registration? In pulumi or outside?
* How can we ensure that what we deploy is instantly fully secure (super tenant is secure) immediately after deployment, and doesn't require manual steps?

These two pieces are related. We can do everything in k8s jobs, but sometimes it might be advantageous to see okapi logs while doing things like bootstrapping the super and registering modules.

**Comment by steveellis** [comment]
I believe all Iris modules have been backported with a fix to the log4j vulnerability but we should also add `LOG4J_FORMAT_MSG_NO_LOOKUPS=true` to the env as well.

**Comment by steveellis** [comment]
Made a note about this here: https://github.com/culibraries/folio/issues/154#issuecomment-1026088787.

**Comment by steveellis** [comment]
Going to close this out:
* I have added the secure-supernenant.py script to the repo.
* I have run it successfully on the deployment.
* Verified that https://folio-iris-okapi.cublcta.com:9130/_/proxy/tenants/cubl/modules no longer returns a list of modules.

I attempted to add `LOG4J_FORMAT_MSG_NO_LOOKUPS` to the deployment, but due to the way that folio-helm handles translating the secrets to the env, it won't be picked up on the pods, even when restarting etc. The env vars are hardcoded to a certain subset. This use wasn't 

So if we're going to continue to use folio-helm we're going to have to rely on the modules to do backporting for log4j. Many key ones have (verified mod-inventory, mod-authtoken, okapi for example), some do not appear to have from their release notes.
